### DISCUSSION TITLE
inhererit substituteFrom from previous Kustomizations

### DISCUSSION THREAD

**Discussion by anthr76** [fluxcd/flux2#1936] [Q&A] [answered]
Wondering if it would be possible to inherit decryption providers and substituteFrom for Kustomizations applied after where it's set.

This functionality would be useful in a heavy depedency tree driven kustomize layout. Without the need to copy/paste

**Comment by stefanprodan** [comment, marked as answer]
You can use a kustomize patch and add all common fields to the Flux Kustomizations, the root one `flux-system` that applies all others will patch them accordingly.

**Reply by anthr76** [reply]
Great idea!
